repository/src/main/java/org/alfresco/repo/security/authority/AuthorityDAOImpl.java

```diff
@@ -378,27 +379,37 @@ public void addAuthority(Collection<String> parentNames, String childName)
         }
     }
 
-    public void createAuthority(String name, String authorityDisplayName, Set<String> authorityZones)
+    @Override
+    public void createAuthority(String name, String authorityDisplayName, Set<String> authorityZones) {
+        createAuthority(name, authorityDisplayName, authorityZones, null);
+    }
+
+    @Override
+    public void createAuthority(String name, String authorityDisplayName, Set<String> authorityZones, Map<QName, Serializable> properties)
     {
-        HashMap<QName, Serializable> props = new HashMap<QName, Serializable>();
+        Map<QName, Serializable> props = new HashMap<>();
         /* MNT-11749 : Alfresco allows to create authorities with different char cases, but disallow duplicates */
         props.put(ContentModel.PROP_NAME, DigestUtils.md5Hex(name));
         props.put(ContentModel.PROP_AUTHORITY_NAME, name);
         props.put(ContentModel.PROP_AUTHORITY_DISPLAY_NAME, authorityDisplayName);
+        if (MapUtils.isNotEmpty(properties))

```

Comment:
Is this checking for `properties == null`? If not then we can just omit the if statement and always putAll the empty map.

Reply:
Good point it actually should check `properties != null`, fixed now